src/components/periodic-report/periodic-report.repository.ts

```diff
@@ -205,15 +205,22 @@ export class PeriodicReportRepository extends DtoRepository<
     return (query: Query) =>
       query.comment`matchCurrentDue()`
         .match([
-          node('baseNode', 'BaseNode', { id: parentId }),
-          relation('out', '', 'report', ACTIVE),
-          node('node', `${reportType}Report`),
-          relation('out', '', 'end', ACTIVE),
-          node('end', 'Property'),
```

Comment:
So still _matching_ on the `endDate`, but now _sorting_ on the `startDate`?

Reply:
Yes still filtering on what's applicable according to the end date and now, but of those candidates now picking (via sort) the by the earliest start date.